shared/src/test/diff/codegen/OptionalParam.mls

```diff
@@ -355,15 +354,24 @@ n.ll
 //│ // Query 1
 //│ globalThis.m = g(1);
 //│ // Query 2
-//│ globalThis.n = m(2);
+//│ globalThis.n = new m(2);
 //│ // Query 3
 //│ res = n.ll;
 //│ // End of generated code
 //│ m
 //│   = [class L]
 //│ n
-//│ Runtime error:
-//│   TypeError: Class constructor L cannot be invoked without 'new'
+//│   = L {}
 //│ res
+//│     = 3
+
+// TODO reject
+// :e
+:re
+class M()
+let mm = new M()
```

Comment:
Why can't we make this work? The logic is the same as the logic to determine whether we generate `extends C.class` or `extends C`.

Reply:
```ts
class M()
let mm = new M()
//│ globalThis.mm = new M.class();
```